src/mpl/include/mpl_atomic_opa.h

```diff
@@ -0,0 +1,218 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+#ifndef MPL_ATOMIC_OPA_H_INCLUDED
+#define MPL_ATOMIC_OPA_H_INCLUDED
+
+#include <stdint.h>
+#include "../../openpa/src/opa_primitives.h"
```

Comment:
Just `#include "opa_primitives.h` should work. The main configure scripts need configure openpa anyway and it will add the path to CPPFLAGS. In general, if a user want to use an external library, the include path is supposed to be pre-configured.

Reply:
True. I will fix it.